net.bartkessels.getit.json

```diff
@@ -0,0 +1,75 @@
+{
+	"app-id": "net.bartkessels.getit",
+	"runtime": "org.gnome.Platform",
+	"runtime-version": "3.24",
+	"sdk": "org.gnome.Sdk",
+	"command": "getit",
+	"finish-args": [
+		"--socket=x11",
+		"--share=ipc",
+		"--socket=wayland",
+		"--share=network",
+		"--filesystem=home",
```

Comment:
What do you need full filesystem access for? If you use GtkFileChooserNative https://developer.gnome.org/gtk3/stable/gtk3-GtkFileChooserNative.html then your app automatically uses the portal to get only the files you user asks for.